examples/solids/problems/cl-problems.h

```diff
@@ -3,12 +3,13 @@
 
 // Problem options
 typedef enum {
-  ELAS_LINEAR = 0, ELAS_SS_NH = 1, ELAS_FSInitial_NH1 = 2, ELAS_FSInitial_NH2 = 3,
-  ELAS_FSCurrent_NH1 = 4, ELAS_FSCurrent_NH2 = 5, ELAS_FSInitial_MR1 = 6
+  ELAS_LINEAR = 0, ELAS_SS_NH = 1, ELAS_FSInitial_NH1 = 2, ELAS_FSInitial_NH1_AD = 3, ELAS_FSInitial_NH2 = 4,
+  ELAS_FSCurrent_NH1 = 5, ELAS_FSCurrent_NH2 = 6, ELAS_FSInitial_MR1 = 7
 } problemType;
 static const char *const problemTypes[] = {"Linear",
                                            "SS-NH",
                                            "FSInitial-NH1",
+                                           "FSInitial-NH1-AD",
```

Comment:
I'd just call this `FSInitial-NH-AD`, personally since we're bypassing the mathematical formulation of the Jacobian.

Reply:
Oh, I see the difference between `FSInitial-NH1` and `FSInitial-NH2` now. Fair point!